apps/meteor/app/emoji/client/emojiParser.ts

```diff
@@ -52,10 +60,11 @@ export const emojiParser = (html: string) => {
 				const { classList } = emojis[i];
 				classList.add('big');
 			}
-			html = checkEmojiOnly.innerHTML;
 		}
 	}
 
+	html = checkEmojiOnly.innerHTML;
```

Comment:
Could you elaborate a bit why this was moved, I couldn't understand the reason only looking at the code

Reply:
Because in the end, all HTML should be put onto this one var... No need to have only inside of a conditional case

Reply:
Originally this was used only to handle the "big emoji" case (one line containing only emojis). With one additional case, better generalize it.